dev-docs/publisher-api-reference.md

```diff
@@ -761,6 +762,73 @@ If a custom adServerTargeting function can return an empty value, this boolean f
+* user privacy - Some publishers may want to opt out of these practices even though it limits their user's values on the open market.
+* security - Publishers may want to control which bidders are trusted to inject sync javascript into their pages.
+
+A simple example of setting the userSync rules:
+{% highlight js %}
+pbjs.userSync = {
+    pixelEnabled: true,   // allow all adapters to drop user-sync pixels
+    syncDelay: 6000       // 6 seconds after the auction
+};
+{% endhighlight %}
+
+In this example, only certain adapters are allowed to drop sync pixels:
+{% highlight js %}
+pbjs.userSync = {
+    enabledBidders: ['abc','xyz'] // only these bidders are allowed to sync
+    pixelEnabled: true,           // and only for pixels
+    syncsPerBidder: 3,            // and no more than 3 pixels at a time
+    syncDelay: 6000,              // 6 seconds after the auction
+};
+{% endhighlight %}
+
+In the last example, the same bidders can drop sync pixels, but the timing will be controlled by the page:
+{% highlight js %}
+pbjs.userSync = {
+    enabledBidders: ['abc','xyz'] // only these bidders are allowed to sync
+    pixelEnabled: true,           // and only for pixels
+    enableOverride: true          // publisher will call pbjs.userSync.syncAll()
+};
+{% endhighlight %}
+
+Here are all the options for userSync control:
+
+{: .table .table-bordered .table-striped }
+| Attribute | Type | Description |
+| --- | --- | --- |
+| syncDelay | integer | The delay in milliseconds for autosyncing once the first auction is run. 3000 by default. |
+| pixelEnabled | boolean | Enables image pixel syncs. |
+| syncsPerBidder | integer | Number of registered syncs allowed per adapter. Default is all. |
+| enabledBidders | array | Array of names of trusted adapters which are allowed to sync users. |
+| enableOverride | boolean | Allows the publisher to manually trigger the user syncs to fire. This prevents autosyncing and exposes the method syncAll() below. |
+| syncAll | function | The page code calls this function to trigger user syncing. Only available if enableOverride is true. |
+
+{: .alert.alert-success :}
+In the near future, there will be options for `iframeEnabled` and `ajaxEnabled`.
+
+#### How it works
+
+The [registerSync()]({{site.baseurl}}/dev-docs/bidder-adaptor.html#step-6-register-user-sync-pixels) function called by the adapter keeps a queue of valid userSync requests. It prevents unwanted sync entries from being placed on the queue:
+
+* Removes undesired sync types. (i.e. enforces the pixelEnabled flag)
+* Removes undesired adapters. (i.e. enforces the enabledBidders option)
+* Makes sure there's not too many queue entries from a given adapter. (i.e. enforces syncsPerBidder)
+
+When syncAll() runs, regardless of whether it's initiated by platform or page, the queue entries are randomized and appended to the bottom of the HTML head tag. If there's no head tag, then they're appended to the end of the body tag.
```

Comment:
Since syncAll() only applies when the publisher chooses to override the automatic syncing, maybe this should read: "When user syncs are run, regardless of whether they are automated by the platform or the page triggers userSync.syncAll(), the queue entries..."